CONTRIBUTING.adoc

```diff
@@ -0,0 +1,96 @@
+= Contributing to Labs CI / CD 
+:toc:
+:toc-title:
+:toc-placement!:
+
+Thanks for checking out the Labs CI / CD guides! We’re excited to hear
+and learn from you. Your experiences will benefit others who read and
+use these guides.
+
+We’ve put together the following guidelines to help you figure out where
+you can best be helpful.
+
+toc::[]
+
+== Types of contributions we’re looking for
+
+This project is used as a starting point for CI/CD tooling by the 
+Red Hat Open Innovation Labs group. It incoporates many technologies
+and is intended for development on OpenShift Container Platform.
+
+Interested in making a contribution? Read on!
+
+== Ground rules & expectations
+
+Before we get started, here are a few things we expect from you (and
+that you should expect from others):
+
+* Be kind and thoughtful in your conversations around this project. We
+all come from different backgrounds and projects, which means we likely
+have different perspectives on "how open source is done". Try to
+listen to others rather than convince them that your way is correct.
+* There is a
+link:./CODE_OF_CONDUCT.adoc[Contributor Code of Conduct]. By participating
+in this project, you agree to abide by its terms.
+* If you open a pull request, please ensure that your contribution
+passes all tests. If there are test failures, you will need to address
+them before we can merge your contribution.
+* When adding content, please consider if it is widely valuable. Please
+don’t add references or links to things you or your employer have
+created as others will do so if they appreciate it.
+
+== Getting set up for local development
+
+Local Development guide is still under construction...
+
+=== Deploying to OpenShift 
+
+The Labs CI / CD stack can be deployed to any OpenShift cluster, and is automated using link:https://github.com/redhat-cop/openshift-applier[OpenShift Applier]. The following commands can be used to install Applier, log in to your OpenShift cluster, and deploy the pipeline.
+
+[source,bash]
+----
+ansible-galaxy install -r requirements.yml -p galaxy
+oc login ...
+ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
+----
+
+This will result in the creation of a pipeline and deployment environments across several projects:
```

Comment:
Maybe a bit clearer here? ci-cd pipeline? jenkins pipeline? ocp pipeline? Is deployment environments the right phrasing? As I read on it becomes clearer. Should Development be lowercase (or promotion by uppercase?

Reply:
same as above. Development is capitalized because it's a proper noun here, i guess? i don't really have a preference. will update